odoo/tools/config.py

```diff
@@ -215,6 +215,8 @@ def _build_cli(self):
                          help="install one or more modules (comma-separated list, use \"all\" for all modules), requires -d")
         group.add_option("-u", "--update", dest="update", type='comma',  metavar="MODULE,...", my_default=[], file_loadable=False,
                          help="update one or more modules (comma-separated list, use \"all\" for all modules). Requires -d.")
+        group.add_option("--reinit", dest="reinit", type='comma', metavar="MODULE,...", my_default=[], file_loadable=False,
```

Comment:
Note: not all modules are reinitializable.

for example, we cannot reinitialize `base` becasue
https://github.com/odoo/odoo/blob/9c1935bf4de8b6db6bcba251208f0f8c9de3df34/odoo/addons/base/data/res_users_data.xml#L20
will remove the admin user from the `base.system_group` and the below error will be raised

https://github.com/odoo/odoo/blob/9c1935bf4de8b6db6bcba251208f0f8c9de3df34/odoo/addons/base/models/res_users.py#L538-L540

The error won't be raised the first time install the `base` because when the `user_admin` is created, there was no `group_system`